string name
geometry_msgs/Pose pose
geometry_msgs/Pose2D pose2D
float32 width_meter
float32 width_pixel
float32 quality
